proposals/static-theme-wizard.md

```diff
@@ -0,0 +1,25 @@
+# AMO Devhub Static Theme Wizard
+
+**Mentor:** Andrew Williamson
```

Comment:
Presumably Andrew is onboard with the commitment for mentoring a GSOC project? Or is this intended more of a joint effort between you and Andrew if it's picked?

Reply:
Good point. No, I didn't speak with Andrew about this. I'll change this to me for now. These are all just drafts anyway, and if Mozilla is accepted into the program we can change these proposals later on.